### PR TITLE
feat: offset radius for section added.

### LINKED ISSUE #1146: Offset radius for each section
Please add support for adding offset radius for each section.

This helps to add offset to the section to obtain result shown in image:

image

### PR DESCRIPTION
**Pull request by ujwalbasnet1**
Adds offset radius to each section

image

Fixes #1146

### CODE REVIEW

**Comment by imaNNeoFighT** [comment]
Hi. Thanks for contributing.
Is there any related issue to this change? If you found any, please mention them.

BTW you need to update [CHANGELOG.md](https://github.com/imaNNeoFighT/fl_chart/blob/master/CHANGELOG.md) and [pie_chart.md](https://github.com/imaNNeoFighT/fl_chart/blob/master/repo_files/documentations/pie_chart.md) files. 
We explained the contribution details [here](https://github.com/imaNNeoFighT/fl_chart/blob/master/CONTRIBUTING.md).

**Comment by ujwalbasnet1** [comment]
@imaNNeoFighT changes made

**Comment by imaNNeoFighT** [comment]
Great!
I suggest you change `offsetRadius` to `offset` or `centerOffset`.
What do you think?

**Comment by ujwalbasnet1** [comment]
@imaNNeoFighT changes made. Thanks

**Comment by imaNNeo** [comment]
Any update?

**Comment by ujwalbasnet1** [comment]
@imaNNeo changes pushed.

File: example/lib/presentation/samples/pie/pie_chart_sample2.dart [hunk 1]
```diff
@@ -148,6 +148,7 @@ class PieChart2State extends State {
             color: AppColors.contentColorGreen,
             value: 15,
             title: '15%',
+            centerOffset: 20,
```
Comment (imaNNeo) [inline example/lib/presentation/samples/pie/pie_chart_sample2.dart:151]: Please revert this change. It does not feel good in our UI/UX.

File: example/lib/presentation/samples/pie/pie_chart_sample3.dart [hunk 2]
```diff
@@ -80,6 +80,7 @@ class PieChartSample3State extends State {
             color: AppColors.contentColorYellow,
             value: 30,
             title: '30%',
+            centerOffset: 30,
```
Comment (imaNNeo) [inline example/lib/presentation/samples/pie/pie_chart_sample3.dart:83]: Please revert this change. It does not feel good in our UI/UX.

**Comment by imaNNeo** [comment]
It is ready to merge. But you need to make some minor changes:

1. Please remove the changes in the examples.
2. Please update the CHANGELOG.md file to write about your feature.
3. Also, don't forget to update the docs in bar_chart.md

Thank you!

**Comment by imaNNeo** [comment]
Don't forget to read the [contributing guideline](https://github.com/imaNNeo/fl_chart/blob/master/CONTRIBUTING.md).
